generators/guestcontainer/index.js

```diff
@@ -99,6 +104,16 @@ module.exports = generators.Base.extend({
             var serviceName = this.props.serviceName;
             var appTypeName = this.projName + 'Type';
             var instanceCount = this.props.instanceCount;
+            if(this.props.portMap != ""){
+                var portMap = this.props.portMap.split(":");
```

Comment:
What id split has some error because of invalid format

Reply:
Added error handling for invalid format